src/diagnostics/diagnostic-structs.md

```diff
@@ -180,8 +180,8 @@ following attributes:
   - Value is a path to an item in `rustc_errors::fluent` for the note's
     message.
     - Defaults to equivalent of `.label`.
-- `#[warn_]` or `#[warn_(slug)]` (_Optional_)
-  - _Applied to `Span` fields._
+- `#[warning]` or `#[warning(slug)]` (_Optional_)
+  - _Applied to struct or `Span`/`()` fields._
```

Comment:
isn't `Span` a struct

Reply:
Technically yes, but here it emphasizes placing it on a struct field of type `Span`.

In case this is ambiguous, it should mean like this:
- struct, or
- `Span`/`()` fields

Reply:
would `(), Span, or other struct types` be accurate

Reply:
ahh, much better